platform/podjs/src/browserPod.ts

```diff
@@ -374,8 +374,8 @@ class BrowserPolyNav implements PolyNav {
                 `polyNav: Keyboard navigation available: ${actionUsage}`
             );
         }
-        this.keyUpListener = ({ key }: any) => {
-            const action = actionKeys[key];
+        this.keyUpListener = (key: KeyboardEvent) => {
```

Comment:
Nit: Would be nice if we could still use destructuring here, e.g. `{ key }: KeyboardEvent` - but I guess that doesn't work?